pyproject.toml

```diff
@@ -84,6 +84,7 @@ filterwarnings = [
   'ignore:setDaemon\(\) is deprecated, set the daemon attribute instead:DeprecationWarning',
   'ignore:TriCutTool is deprecated::',
   'ignore:Cut3dTool is deprecated::',
+  'ignore: *scattermapbox* is deprecated:DeprecationWarning',
```

Comment:
Not what I meant. We should actually change the implementation to work with the new API. Otherwise it will just break without warning.